compiler-core/src/language_server/compiler.rs

```diff
@@ -124,6 +130,12 @@ where
             }
             // Create the source information
             let path = module.src_path.to_string();
+            // strip canonicalised windows prefix
+            #[cfg(target_family = "windows")]
```

Comment:
ran into this while testing on my other computer. Only causes issues with local deps (I'm guessing due to calling canonicalise similar to https://github.com/gleam-lang/gleam/pull/2842). Could pull this out into a separate thing if preferred

Reply:
Also, not sure if you want me to pull this out if this pr won't get into the 1.1 release (I'm fine with whatever but lmk what you think. I think the fix here really should be in 1.1 but the rest makes sense to leave out)